config/pkg/pldconf/startup.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2024 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pldconf
+
+import (
+	"github.com/kaleido-io/paladin/config/pkg/confutil"
+)
+
+type StartupConfig struct {
+	BlockchainConnectRetry RetryConfigWithMax `json:"blockchainConnectRetry"`
+}
+
+var StartupConfigDefaults = StartupConfig{
```

Comment:
This is the replacement of the logic added to `ethclient` in https://github.com/LF-Decentralized-Trust-labs/paladin/commit/e81df2f743e5b685a0ed13fe5434c87524318b32 as part of #403